Mobile menu

ahmedadel56 opened the pull request:
In this branch, I tried to add a mobile side menu and I hope you like it

Davidosky007 commented:
## Hi @ahmedadel56 👋, 

Good job so far! 💪 
* Nice commit message ✔️ 
* descriptive commit messages ✔️ 
* professional PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Kindly add the linter files properly 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

ahmedadel56 commented:
> ## Hi @ahmedadel56 ,
> Good job so far! 
> 
> * Nice commit message heavy_check_mark
> * descriptive commit messages heavy_check_mark
> * professional PR heavy_check_mark
>   There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Required Changes 
> * Kindly add the linter files properly
>   _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

please tell me where are comments to check

ahmedadel56 commented:
please tell me more details cause I don't see any comments to check

On Tue, Sep 7, 2021 at 7:36 PM David Bassey ***@***.***>
wrote:

> Hi @ahmedadel56 <https://github.com/ahmedadel56> 👋,
>
> Good job so far! 💪
>
>    - Nice commit message ✔️
>    - descriptive commit messages ✔️
>    - professional PR ✔️
>    There are some issues that you still need to work on to go to the next
>    project but you are almost there!
>
> Required Changes ♻️
>
>    - Kindly add the linter files properly
>    *Check the comments under the review.*
>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ahmedadel56/ahmedadel56.github.io/pull/18#issuecomment-914495100>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKJNT75IHKHCSE42UMM4U7TUAZETXANCNFSM5DSTSM6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

Davidosky007 commented:
> please tell me more details cause I don't see any comments to check
> […](#)
> On Tue, Sep 7, 2021 at 7:36 PM David Bassey ***@***.***> wrote: Hi @ahmedadel56 <https://github.com/ahmedadel56> 👋, Good job so far! 💪 - Nice commit message ✔️ - descriptive commit messages ✔️ - professional PR ✔️ There are some issues that you still need to work on to go to the next project but you are almost there! Required Changes ♻️ - Kindly add the linter files properly *Check the comments under the review.* Optional suggestions *Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.* Cheers and Happy coding!👏👏👏 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. *Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.* ------------------------------ *As described in the Code reviews limits policy <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.* — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#18 (comment)](https://github.com/ahmedadel56/ahmedadel56.github.io/pull/18#issuecomment-914495100)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AKJNT75IHKHCSE42UMM4U7TUAZETXANCNFSM5DSTSM6A> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

Kindly check your linter files they are not properly created (.github/workflows) [Use this config ](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

ahmedadel56 commented:
> > please tell me more details cause I don't see any comments to check
> > […](#)
> > On Tue, Sep 7, 2021 at 7:36 PM David Bassey _**@**_.***> wrote: Hi @ahmedadel56 https://github.com/ahmedadel56 wave, Good job so far! muscle - Nice commit message heavy_check_mark - descriptive commit messages heavy_check_mark - professional PR heavy_check_mark There are some issues that you still need to work on to go to the next project but you are almost there! Required Changes recycle - Kindly add the linter files properly _Check the comments under the review._ Optional suggestions _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._ Cheers and Happy coding!clapclapclap Feel free to leave any questions or comments in the PR thread if something is not 100% clear. _Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._ ------------------------------ _As described in the Code reviews limits policy https://microverse.zendesk.com/hc/en-us/articles/1500004088561 you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form https://airtable.com/shrQAqnBwek5a0O0s._ — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#18 (comment)](https://github.com/ahmedadel56/ahmedadel56.github.io/pull/18#issuecomment-914495100)>, or unsubscribe https://github.com/notifications/unsubscribe-auth/AKJNT75IHKHCSE42UMM4U7TUAZETXANCNFSM5DSTSM6A . Triage notifications on the go with GitHub Mobile for iOS https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675 or Android https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub.
> 
> Kindly check your linter files they are not properly created (.github/workflows) [Use this config ](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

I fixed them so please check it again

ahmedadel56 commented:
I fixed it so please check it again

On Tue, Sep 7, 2021 at 7:49 PM David Bassey ***@***.***>
wrote:

> please tell me more details cause I don't see any comments to check
> … <#m_-874076742588733203_>
> On Tue, Sep 7, 2021 at 7:36 PM David Bassey *@*.***> wrote: Hi
> @ahmedadel56 <https://github.com/ahmedadel56>
> https://github.com/ahmedadel56 👋, Good job so far! 💪 - Nice commit
> message ✔️ - descriptive commit messages ✔️ - professional PR ✔️ There
> are some issues that you still need to work on to go to the next project
> but you are almost there! Required Changes ♻️ - Kindly add the linter
> files properly *Check the comments under the review.* Optional
> suggestions *Every comment with the [OPTIONAL] prefix is not crucial
> enough to stop the approval of this PR. However, I strongly recommend you
> to take them into account as they can make your code better.* Cheers and
> Happy coding!👏👏👏 Feel free to leave any questions or comments in the
> PR thread if something is not 100% clear. *Please, do not open a new Pull
> Request for re-reviews. You should use the same Pull Request submitted for
> the first review, either valid or invalid unless it is requested otherwise.*
> ------------------------------ *As described in the Code reviews limits
> policy https://microverse.zendesk.com/hc/en-us/articles/1500004088561
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form https://airtable.com/shrQAqnBwek5a0O0s
> <https://airtable.com/shrQAqnBwek5a0O0s>.* — You are receiving this
> because you were mentioned. Reply to this email directly, view it on GitHub
> <#18 (comment)
> <https://github.com/ahmedadel56/ahmedadel56.github.io/pull/18#issuecomment-914495100>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AKJNT75IHKHCSE42UMM4U7TUAZETXANCNFSM5DSTSM6A
> . Triage notifications on the go with GitHub Mobile for iOS
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> or Android
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> .
>
> Kindly check your linter files they are not properly created
> (.github/workflows) Use this config
> <https://github.com/microverseinc/linters-config/tree/master/html-css-js>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ahmedadel56/ahmedadel56.github.io/pull/18#issuecomment-914503306>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKJNT7YGKCQWDJ7AMTNNX5TUAZGELANCNFSM5DSTSM6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

SpaYco reviewed (commented):
Hi @ahmedadel56  👋,

_This is an additional review as a result of the second opinion request._

Reviewers would do their best to help you with your questions, however, they are not supposed to review again unless you ask for another review through your dashboard

Please go ahead and request another and a CR would review it ASAP.

Keep rocking 🚀!

SajjadAhmad14 commented on dynamic.js:
```diff
@@ -0,0 +1,50 @@
+// creating mobile menue elements
```
Please add linter config from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link for html+css+js.

ahmedadel56 replied:
> Please add linter config from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link for html+css+js. It's because you have started to use javaScript in your project.

I already did it in .eslintrc.JSON file xo please check it

SajjadAhmad14 reviewed (changes requested):
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Adel Ramadan, 
 ### Thank you for working on changes suggested previously:ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

ahmedadel56 commented:
I already added js linter config file and I fixed the errors so please
give me more details to know what you are talking about


On Tue, Sep 7, 2021 at 10:00 PM Sajjad Ahmad ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> Required Changes ♻️
>
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
>
> Hi Adel Ramadan, Thank you for working on changes suggested previously👌 To
> Improve 🙏
>
>    - Please check inline comments under the review
>
> Optional suggestions
>
> *Comments with the [OPTIONAL] prefix are not a must to implement however,
> It's strongly recommended to work on them as they can make your code
> better.*
> Happy coding!👏👏👏 Please leave any questions or comments in the PR if
> you have any doubts or unclarity!
>
> *Note: Please, do not open a new Pull Request for re-reviews. You should
> use the same Pull Request submitted for the first review, either valid or
> invalid unless it is requested otherwise.*
> ------------------------------
>
> As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have
> a limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.
> ------------------------------
>
> In dynamic.js
> <https://github.com/ahmedadel56/ahmedadel56.github.io/pull/18#discussion_r703791838>
> :
>
> > @@ -0,0 +1,50 @@
>
> +// creating mobile menue elements
>
>
> Please add linter config from this
> <https://github.com/microverseinc/linters-config/tree/master/html-css-js>
> link for html+css+js.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ahmedadel56/ahmedadel56.github.io/pull/18#pullrequestreview-748348620>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKJNT767DT46YG2UA4DYYP3UAZVOTANCNFSM5DSTSM6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

SpaYco commented:
@ahmedadel56 you're missing ESLint, use this file
https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml

ahmedadel56 commented:
> @ahmedadel56 you're missing ESLint, use this file
> https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml

okay fixed

ahmedadel56 commented:
Hey Aziz
thank you very much for reviewing my code, I just want to tell you that I
fixed the linter.yml file so please check it again

On Tue, Sep 7, 2021 at 10:17 PM Aziz Mejri ***@***.***> wrote:

> @ahmedadel56 <https://github.com/ahmedadel56> you're missing ESLint, use
> this file
>
> https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ahmedadel56/ahmedadel56.github.io/pull/18#issuecomment-914595353>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKJNT75JF5SDDUGVBQ7IUKTUAZXMNANCNFSM5DSTSM6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

SpaYco commented:
@ahmedadel56 it looks good now, it should be approved, just ask for another review :)

barackm reviewed (approved):
Hi @ahmedadel56, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 

![](https://media.giphy.com/media/vmtxnxveVUodG/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
